apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -14,12 +14,14 @@ import { ApiPathSelectionSection } from '@components/config-screen/ApiPathSelect
 import { AuthenticationSection } from '@components/config-screen/AuthenticationSection/AuthenticationSection';
 import { copies } from '@constants/copies';
 import { actions } from '@constants/enums';
+import { ConfigPageProvider } from '@contexts/ConfigPageProvider';
 
 const ConfigScreen = () => {
   const [isTokenValid, setIsTokenValid] = useState(false);
   const [contentTypes, setContentTypes] = useState<ContentType[]>([]);
   const [projects, setProjects] = useState<Project[]>([]);
   const [apiPaths, setApiPaths] = useState<ApiPath[]>([]);
+  const [isAppConfigureCalled, setIsAppConfigureCalled] = useState(false);
```

Comment:
yeah this may be a naming mishap on my end! but this state essentially is allowing the Preview path component to know that the user has "saved/installed" the current configuration in order to render that warning message. We only know this at the top level when `onConfigure` is called and the preview path warning message only appears on save/install. So this initial state is false, and then is an ephemeral true, until the user then interacts with the Content type/preview path section, which then calls the `clearIsAppConfigureCalled` function to reset this state.

Reply:
This has been adjusted, to render as `true` in the beginning 👍